ipi/utils/io/inputs/io_xml.py

```diff
@@ -533,10 +533,10 @@ def write_tuple(data):
 def write_float(data):
     """Writes a formatted string from a float.
 
-    Floats are printed out in exponential format, to 8 decimal places and
-    filling up any spaces under 16 not used with spaces.
+    The default format is exponential to 10 decimal places and
+    filling up any spaces under 18 not used with spaces.
 
-    For example 1.0 --> '  1.00000000e+00'
+    For example 1.0 --> '  1.0000000000e+00'
```

Comment:
I would do the % 17.9e format so it's space-neutral WRT the current format

Reply:
Not sure what you mean. 17.9 isn't neutral with current 16.8 anyways. I guess you mean 16.9 with sacrifice of one whitespace? But worth noticing that there can be a minus sign which takes place of one of two whitespaces. I wouldn't leave only one space because of that.